### PR TITLE
Add STACK libraries for qtype_stack version 2022082900

### PR DESCRIPTION
**Pull request by ngandrass**
Added all STACK libraries and `maximalocal.mac.template` for the latest STACK 4.4.1 (2022082900) release.

### CODE REVIEW

**Comment by ngandrass** [comment]
Oh, and I totally forgot to mention that this patch was successfully tested ;)

image
image

File: stack/2022082900/maximalocal.mac.template [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* ***********************************************************************/
+/* This file is automatically generated at installation time.            */
+/* The purpose is to transfer configuration settings to Maxima.          */
+/* Hence, you should not edit this file.  Edit your configuration.       */
+/* This file is regularly overwritten, so your changes will be lost.     */
+/* ***********************************************************************/
+
+/* File generated on July 15, 2022, 4:29 pm */
```
Comment (Lennart-Kramer) [inline stack/2022082900/maximalocal.mac.template]: Have you ensured that the new maximalocal template (as shown on the healthcheck page) is the same as the old one (except for the templated variables)? Sometimes there can be some changes in the units.

**Review by Lennart-Kramer — CHANGES_REQUESTED** [review]
Thanks for updating the version so quickly!

File: stack/2022082900/maximalocal.mac.template (hunk 1 above)
Comment (ngandrass) [reply stack/2022082900/maximalocal.mac.template]: Yes. I took the template generated by the STACK healthcheck page and replaced all fixed paths with the variables. I just forgot to update the top comment block. I'll push an update ASAP.

**Review by Lennart-Kramer — APPROVED** [review]

**Pull request merged**